addons/mail/static/src/new/web/chat_window/chat_window_container.scss

```diff
@@ -2,7 +2,3 @@
     transform: translate(50%, -50%);
     z-index: $zindex-tooltip; // on top of bootstrap dropup menu
 }
-
-.o-mail-chat-window-hidden-menu-windowCounter {
-    user-select: none;
```

Comment:
I have no idea what these lines do and why it's ok to remove them.

Reply:
It's been replaced by classname equivalent `user-select-none`